EMacco/abarian-backend
# #170468625 Get user address

#### What does this PR do?
This PR allows a user to be able to fetch a list of all his saved addresses

#### Description of Task to be completed?
- return an array of user's addresses

#### How should this be manually tested?
- checkout to this branch 
- make a get request to `/addresses`
- an array of addresses should be returned

#### Any background context you want to provide?
- you need to be logged in

#### What are the relevant pivotal tracker stories?
[#170468625](https://www.pivotaltracker.com/story/show/170468625)

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
